integration/fuse/src/main/java/alluxio/fuse/AlluxioFuse.java

```diff
@@ -162,55 +163,44 @@ public static FuseUmountable launchFuse(FileSystemContext fsContext, FileSystem
     String mountPoint = conf.getString(PropertyKey.FUSE_MOUNT_POINT);
     Path mountPath = Paths.get(mountPoint);
     String[] optimizedMountOptions = optimizeAndTransformFuseMountOptions(conf);
-    try {
-      if (!Files.exists(mountPath)) {
-        LOG.warn("Mount point on local filesystem does not exist, creating {}", mountPoint);
+    if (!Files.exists(mountPath)) {
+      LOG.warn("Mount point on local filesystem does not exist, creating {}", mountPoint);
+      try {
```

Comment:
nit: there are FileUtils which wrap java files functions and deal with exceptions. See if we can reuse that or improve utils.

Reply:
Took a look at the FileUtils, looks like it will be better to maintain the current logics for two reasons:
1) Avoid creating the same Path objects repeatedly
2) give clear error message. Instead of "Failed to create directory XXX" we give error message that "Failed to create Fuse mount point XXX" then users know we fail to create the FUSE mount point instead of an arbitrary directory